new-app/src/App/runGame/RunGame.tsx

```diff
@@ -15,11 +15,14 @@ const RunGame = () => {
   }
 
   return (
-    <div>
-      <h1>Nordle</h1>
-      <RandomNumber  {...RandomNumberProps} />
-      <OtpInputGuess guess={guess} setGuess={setGuess} />
-      <Hints guess={guess} randomNumber={randomNumber} />
+    <div className="flex justify-center">
+      <div className="flex flex-col bg-back-charcoal border-8 border-hacky-green rounded-3xl p-16">
+        <h1>Nordle</h1>
+        <RandomNumber  {...RandomNumberProps} />
```

Comment:
I'd probably inline these props. We usually only use this style for 3rd party components that have dozens of props that need to be passed through.